sequencing-server/src/routes/seqjson.ts

```diff
@@ -0,0 +1,461 @@
+
+export const seqjsonRouter = express.Router();
+
+/**
+ * Generate a sequence JSON from a sequence standalone file
+ *
+ * @deprecated Use `/bulk-get-seqjson-for-sequence-standalone` instead
+ */
+seqjsonRouter.post('/get-seqjson-for-sequence-standalone', async (req, res, next) => {
+  const commandDictionaryID = req.body.input.commandDictionaryID as number;
+  const edslBody = req.body.input.edslBody as string;
+
+  let commandTypes;
+  try {
+    const context: Context = res.locals['context'];
+    commandTypes = await context.commandTypescriptDataLoader.load({ dictionaryId: commandDictionaryID });
+  } catch (e) {
+    res.status(500).json({
+      message: 'Error loading command dictionary',
+      cause: (e as Error).message,
+    });
+    return next();
+  }
+
+  const result = Result.fromJSON(
+    await (piscina.run(
+      {
+        edslBody,
+        commandTypes,
+      },
+      { name: 'executeEDSL' },
+    ) as ReturnType<typeof executeEDSL>),
+  );
+
+  if (result.isErr()) {
+    res.json({
+      status: FallibleStatus.FAILURE,
+      seqJson: null,
+      errors: result.unwrapErr(),
+    });
+  }
+
+  res.json({
+    status: FallibleStatus.SUCCESS,
+    seqJson: result.unwrap(),
+    errors: [],
+  });
+
+  return next();
+});
+
+/** Generate multiple sequence JSONs from multiple sequence standalone files */
+seqjsonRouter.post('/bulk-get-seqjson-for-sequence-standalone', async (req, res, next) => {
+  const inputs = req.body.input.inputs as { commandDictionaryId: number; edslBody: string }[];
+
+  const context: Context = res.locals['context'];
+
+  const results = await Promise.all(
+    inputs.map(async ({ commandDictionaryId, edslBody }) => {
+      let commandTypes: string;
+      try {
+        commandTypes = await context.commandTypescriptDataLoader.load({ dictionaryId: commandDictionaryId });
+      } catch (e) {
+        return {
+          status: FallibleStatus.FAILURE,
+          seqJson: null,
+          errors: new Error('Error loading command dictionary', { cause: e }),
+        };
+      }
+
+      const result = Result.fromJSON(
+        await (piscina.run(
+          {
+            edslBody,
+            commandTypes,
+          },
+          { name: 'executeEDSL' },
+        ) as ReturnType<typeof executeEDSL>),
+      );
+
+      if (result.isErr()) {
+        return {
+          status: FallibleStatus.FAILURE,
+          seqJson: null,
+          errors: result.unwrapErr(),
+        };
+      } else {
+        return {
+          status: FallibleStatus.SUCCESS,
+          seqJson: result.unwrap(),
+          errors: [],
+        };
+      }
+    }),
+  );
+
+  res.json(results);
+
+  return next();
+});
+
+/**
+ * Get the sequence JSON for a sequence based on seqid and simulation dataset id
+ *
+ * @deprecated Use `/bulk-get-seqjson-for-seqid-and-simulation-dataset` instead
+ */
+seqjsonRouter.post('/get-seqjson-for-seqid-and-simulation-dataset', async (req, res, next) => {
```

Comment:
Should we remove this endpoint now so we are doing this refactor? 

You will have to remove the action:
https://github.com/NASA-AMMOS/aerie/blob/7dadcf0b2354ad06da9037530447be2d21cfdc0f/deployment/hasura/metadata/actions.yaml#L50

Reply:
That will require a UI change as well just FYI

Reply:
Maybe we can wait until the UI changes to using the newer bulk endpoints and then we can remove them?